appinfo/routes.php

```diff
@@ -52,6 +52,73 @@
 		'verb' => 'POST'
 	],
 
+	[
```

Comment:
There is no need to have those as regular routes, since Nextcloud can just reuse the OCS endpoints.